charts/helm-dashboard/values.yaml

```diff
@@ -20,6 +20,8 @@ image:
   # Specifies the exact image digest to pull.
   digest: ""
   imagePullSecrets: []
+  # Flag for using an image with fixed base path /helm-dashboard
+  basePath: false
```

Comment:
Can we make this parameter string instead of boolean? This way user would be able to set any base path he wants, not only `helm-dashboard`. More flexible.